scripts/nuke.sh

```diff
@@ -0,0 +1,13 @@
+rm -r node_modules
+
+for d in packages/*
+do
+    echo $d
+    rm -r $d/node_modules $d/dist $d/.turbo
```

Comment:
This won't work on windows machines, we'll probably need to use rimraf sometime in the future.
Just leaving a comment as we don't need to tackle this soon.